src/core/DOMEvent.ts

```diff
@@ -35,9 +40,17 @@ const DOMEvent = (() => {
      * @param useCapture Boolean
      * @return null
      */
-    this.on = function (obj, type, listener, context, useCapture) {
+    // @ts-ignore
```

Comment:
These `ts-ignore` statements are kinda concerning. I suppose that's here to disable the typechecking?